src/powershell-native/nativemsh/pwrshplugin/entrypoints.cpp

```diff
@@ -307,11 +310,15 @@ DWORD ReportOperationComplete(WSMAN_PLUGIN_REQUEST *requestDetails, DWORD errorC
 
     DWORD result = EXIT_CODE_SUCCESS;
     PWSTR pwszErrorMessage = NULL;
-    GetFormattedErrorMessage(&pwszErrorMessage, errorCode);
+    if (GetFormattedErrorMessage(&pwszErrorMessage, errorCode) == 0)
+    {
+        swprintf_s(szMessage,_countof(szMessage), L"ReportOperationComplete: %d", errorCode);
+        pwszErrorMessage = szMessage;
```

Comment:
Perhaps for readability of the code, it would be better to change this to call WSManPluginOperationComplete with szMessage and else use pwszErrorMessage?

This would also remove the change on line 321 below.

Reply:
Fixed.